flexget/utils/imdb.py

```diff
@@ -166,7 +168,15 @@ def search(self, name):
             movie = {}
             additional = re.findall(r'\((.*?)\)', row.text)
             if len(additional) > 0:
-                movie['year'] = additional[-1]
+                try:
+                    int(additional[-1])
```

Comment:
isdigit() is also less resource intensive. since all it does is do a try int(string)